### DISCUSSION TITLE
407

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#179] [Comments]
# 407

Intro a 링크를 통한 파일 다운로드를 제공 하고 있었는데, 개선이 필요했습니다. 전체 데이터를 poi를 활용해 엑셀 파일을 생성 한 뒤에 반출을 해 주는 과정인데, 데이터가 클 수록 파일을 작성하는데 워낙 시간이 오래 걸리다보니 기다리는동안 사용자 경험이 너무 좋지 않았습니다. 그래서 파일 다운로드 과정을 AJAX를 이용해 비동기로 요청하고, 요청을 기다리는 동안 waitMe 를 이용해 로딩바를 보여주는 식으로 개선을 하기로 했습니다. https://github.com/vadimsva/waitMe AJAX 이를 위해 jQuery AJAX를 활용해 코드를 구현 해 보았습니다. $(&#39;#export&#39;).on(&#39;click&#39;, function () { var wait = $(th..

https://shanepark.tistory.com/407

**Comment by neru0501** [comment]
와 정말 궁금했던 내용이었는데 드디어 찾았네요 감사합니다!
